--- origsrc/gramps-3.0.1/configure.in	2008-05-17 15:30:39.000000000 -0500
+++ src/gramps-3.0.1/configure.in	2008-06-17 21:09:16.796875000 -0500
@@ -87,7 +87,7 @@
 rm -f conftest.out conftest.py
 if test NO = "$has_pygtk"
 then
-   AC_MSG_ERROR([
+   AC_MSG_WARN([
 
 **** The python bindings for gtk 2.4 (pygtk2>=2.10.0) could not be found.])
 fi
@@ -111,7 +111,7 @@
 rm -f conftest.out conftest.py
 if test NO = "$has_pygnome"
 then
-   AC_MSG_ERROR([
+   AC_MSG_WARN([
 **** The python bindings for gnome2 (gnome2-python) could not be found.])
 fi
 AC_MSG_RESULT($has_pygnome)
@@ -133,7 +133,7 @@
 rm -f conftest.out conftest.py
 if test NO = "$has_pyglade"
 then
-   AC_MSG_ERROR([
+   AC_MSG_WARN([
 
 **** The python bindings for glade (pygtk2-libglade) could not be found.])
 fi
